tests/test_glue.py

```diff
@@ -235,3 +284,23 @@ def test_jobs_delete(self):
         client = session_factory().client("glue")
         jobs = client.get_jobs()["Jobs"]
         self.assertFalse(jobs)
+
+
+class TestGlueCrawlers(BaseTest):
+
+    def test_crawlers_delete(self):
+        session_factory = self.replay_flight_data("test_glue_crawler_delete")
+        p = self.load_policy(
+            {
+                "name": "glue-crawler-delete",
+                "resource": "glue-crawler",
+                "filters": [{"Name": "test"}],
+                "actions": [{"type": "delete"}],
+            },
+            session_factory=session_factory,
+        )
+        resources = p.run()
+        self.assertEqual(len(resources), 1)
+        client = session_factory().client("glue")
+        crawlers = client.get_crawlers()["Crawlers"]
+        self.assertFalse(crawlers)
```

Comment:
technically the assertion here should be that the test crawler no longer exists, not that there aren't any more crawlers